src/borg/platform/linux.pyx

```diff
@@ -72,8 +72,11 @@ BSD_TO_LINUX_FLAGS = {
 
 
 def set_flags(path, bsd_flags, fd=None):
-    if fd is None and stat.S_ISLNK(os.lstat(path).st_mode):
```

Comment:
@enkore do you remember why links were skipped here?

the open() following has O_NOFOLLOW, so it should actually work on the symlink itself (if we have one).

Reply:
i tried on linux/ext4, lsattr/chattr not supported on a symlink.
the question now is whether that is fs specific or the case in general on linux.

Reply:
O_NOFOLLOW makes the open fail if it is attempting to open a symlink. The check here would simply avoid that error, but it is still required due to the race condition of the check.

Reply:
ok, i added the symlink check, to both get_flags and set_flags.

Reply:
ok, guess I am done here? collapse / merge?